luma/lcd/device.py

```diff
@@ -757,3 +760,234 @@ def contrast(self, value):
         """
         assert(0 <= value <= 255)
         self.command(0x81, value >> 2)
+
+
+class hd44780(parallel_device, character):
+    """
+    Driver for a HD44780 style LCD display.  This class provides a ``text``
+    property which can be used to set and get a text value, which will be
+    rendered to the display's screen using the display's built-in font.
+
+    :param serial_interface: The serial interface (usually a
+        :py:class:`luma.core.interface.serial.parallel` instance) to delegate
+        sending data and commands through.
+    :param width: The number of characters that can be displayed on a line.
+    :type width: int
+    :param height: The number of lines the display supports.
+    :type height: int
+    :param undefined: character to use if a requested character is not in the
+        font tables
+    :type undefined: str
+    :param selected_font: the font table appropriate for the model of display you
+        are using.  The hd44780 normally comes in a version with font A00 (ENGLISH_JAPANESE) or A02 (ENGLISH_EUROPEAN).  You can provide either
```

Comment:
break this line on around 80-100 chars (as well as any others).

Reply:
Changed as suggested